File: homemarket-domain/src/main/java/com/plateer/homemarket/entity/Member.java
package com.plateer.homemarket.entity;

import java.util.List;

import com.plateer.homemarket.entity.enumType.Authority;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class Member {
	
	private String memberId;
	private String name;
	private String password;
	private String email;
	private String joinDate;
	private List<Address> address;
	private Authority authority;

}

File: vue/src/model/Member.js
class Member {
    constructor(memberId, name, password, email, addresses, joinDate, roles){
        this.memberId = memberId;
        this.name = name;
        this.password = password;
        this.addresses = addresses;
        this.email = email;
        this.joinDate = joinDate;
        this.roles =  roles;

    }
}

export default Member;